mxd/README.md

```diff
@@ -122,8 +122,8 @@ assets, policies and contract definitions.
 In order to check that the connectors were deployed successfully, please execute the following commands in a shell:
 
 ```shell
-curl -X GET http://localhost/bob/health/api/check/liveness
-curl -X GET http://localhost/alice/health/api/check/liveness
+curl -X GET http://localhost/bob/health/api/check/liveness | jq
```

Comment:
lets be careful with piping to `jq` by default. some people might not have it installed (e.g. WSL) and may not understand what's happening.

Reply:
@paullatzelsperger we had the suggestion at the TractusX Community days, because we use it in certain parts of the tutorial anyway. Either completely or not at all, right?

Reply:
Maybe we should add `jq` as an optional prerequisite, like we did it with `stern`.

Reply:
As @florianrusch suggested: it is not needed for anything, other than for pretty-printing. I would put a comment somewhere, that "tools like `jq` can be used to pretty-print output", but we will not require it to be installed.

Reply:
@awellnitz-materna as I said in my previous comment, `jq` should become an optional dependency, thus removing it from the `curl` command.

Reply:
@paullatzelsperger i have a slightly different view than you.
In the README, jq was already used in the requests before, why should we remove it now?
It offers added value, especially for the tutorial.

Here I stick to my statement: Either we should remove it completely or we should use it for all JSON outputs.

Reply:
remove it completely then.